creator-node/src/index.js

```diff
@@ -54,6 +54,9 @@ const startApp = async () => {
   if (!delegateOwnerWallet || !delegatePrivateKey) {
     exitWithError('Cannot startup without delegateOwnerWallet and delegatePrivateKey')
   }
+  if (!config.get('creatorNodeEndpoint')) {
+    logger.warning('WARNING - Missing creatorNodeEndpoint configuration')
```

Comment:
i think at this point we should require this. external sp's should have this filled in just like they have the delegate wallet